api/v1alpha1/hostedcluster_types.go

```diff
@@ -13,8 +13,9 @@ func init() {
 
 const (
 	// AuditWebhookKubeconfigKey is the key name in the AuditWebhook secret that stores audit webhook kubeconfig
-	AuditWebhookKubeconfigKey          = "webhook-kubeconfig"
-	DisablePKIReconciliationAnnotation = "hypershift.openshift.io/disable-pki-reconciliation"
+	AuditWebhookKubeconfigKey           = "webhook-kubeconfig"
+	DisablePKIReconciliationAnnotation  = "hypershift.openshift.io/disable-pki-reconciliation"
+	IdentityProviderOverridesAnnotation = "hypershift.openshift.io/identity-provider-overrides"
```

Comment:
I would expect the override annotation to specify which identity provider it's overriding (in case you support adding other idps later). Something like:
`idpoverrides.hypershift.openshift.io/[idp-name]=[blah]`

Reply:
I see: I like that as well will adjust to that.......

Reply:
adjusted